src/store/localization/reducers.tsx

```diff
@@ -1117,12 +1118,20 @@ const initialState = {
 	}),
 	"publishLevel": new LocalizedStrings({
 		"en": {
-			"none": "Not Published - not visible in publishing app",
+			"none": "Not Published - not visible outside this project",
 			"beta": "Beta - visible on the Akuo website when using the beta parameter (not available in the app)",
 			"betalink": "https://akuobible.org/?beta=true",
 			"public": "Public - visible to everyone using Akuo website or app",
 		}
 	}),
+	"publishTo": new LocalizedStrings({
+		"en": {
+			"Akuo": "Akuo",
+			"Aquifer": "Aquifer",
+			"Internalization": "Internalization Resources for other projects",
+			"OBTHelps": "OBT Helps website",
```

Comment:
There are two different kinds of things going on here. Internationalization is a word that needs translating. Akuo, Aquifer, and OBTHelps are brand names that should occur in the code and not be given to the user to translate. We can hard code them into strings somewhere if you like but they shouldn't be part of the localization file.